package.json

```diff
@@ -1748,11 +1748,20 @@
           ],
           "markdownDescription": "Additional trigger characters for intellisense of LaTeX documents. You must reload VSCode to take into account a change in this configuration."
         },
-        "latex-workshop.intellisense.snippets.trigger.latex": {
+        "latex-workshop.intellisense.atSuggestion.trigger.latex": {
```

Comment:
Should not change the name. We had better add `intellisense.atSuggestion.trigger.latex` as a new entry, and set `markdownDeprecationMessage` to `intellisense.snippets.trigger.latex`.

- https://code.visualstudio.com/api/references/contribution-points
-  #2654

Reply:
Thanks for reminding me of #2654. Btw, it seems that calling 
https://github.com/James-Yu/LaTeX-Workshop/blob/2bd72a13cecf79c6be4f3748945acad539cb8896/src/components/configuration.ts#L16 is useless as https://github.com/James-Yu/LaTeX-Workshop/blob/4b418f4c02913af750423226695a879c61a0a59f/src/components/configurationlib/deprecated.ts#L13 is always empty.

Maybe, it is time to add a first setting to `deprecatedConfigurations`?

Reply:
#3209